### PR TITLE
DO-168: Add retention policy

### PR DESCRIPTION
**Pull request by siameseoriental**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,55 +2,69 @@
...
+    && rm -rf /var/lib/apt/lists/*
 
 RUN curl --location --output /usr/local/bin/mcli "https://dl.min.io/client/mc/release/linux-amd64/mc" && \
     chmod +x /usr/local/bin/mcli
 RUN mcli --version
 
+# PosgeSQL related variables
 ENV POSTGRES_DB="**None**" \
-    POSTGRES_HOST="**None**" \
+    POSTGRES_HOST="db" \
```
Comment (hxmn) [inline Dockerfile]: Кажется, осталось после тестирования

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,38 +0,0 @@
-<!--
```
Comment (hxmn) [inline CHANGELOG.md]: Why have you removed changelog?

File: compose-example/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    env_file:
+      - variables.env
+    depends_on:
+      - db
+
+  redis:
+    image: redis:7
+
+  web:
+    image: ........
```
Comment (hxmn) [inline compose-example/docker-compose.yml]: Why do we need those here?

File: compose-example/variables.env [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Mandatory values
+S3_ENDPOINT=https://io.some-domain.com
+S3_ACCESS_KEY=${S3_ACCESS_KEY_DB_BACKUPS}
+S3_SECRET_KEY=${S3_SECRET_KEY_DB_BACKUPS}
+POSTGRES_DB=some_db_name
+POSTGRES_USER=some_db_username
+POSTGRES_PASSWORD=some_db_user_password
+
+# Mandatory only in case of scheduled usage
+S3_BUCKET=bucket_name/project_name/stage_branch/ # Including name of the bucket and (optional) the rest of directory structure
```
Comment (hxmn) [inline compose-example/variables.env]: Looks like a path

**Review by hxmn — CHANGES_REQUESTED** [review]

File: compose-example/variables.env (hunk 4 above)
Comment (siameseoriental) [reply compose-example/variables.env]: Fixed

File: compose-example/docker-compose.yml (hunk 3 above)
Comment (siameseoriental) [reply compose-example/docker-compose.yml]: Removed

File: CHANGELOG.md (hunk 2 above)
Comment (siameseoriental) [reply CHANGELOG.md]: Occasionally maybe. Got it back, added changes I have made.

File: Dockerfile (hunk 1 above)
Comment (siameseoriental) [reply Dockerfile]: Fixed

**Review by hxmn — APPROVED** [review]

**Pull request merged**